### ISSUE TITLE
[hack] Add opaque operators for all operators that Glenside currently doesn't support

### ISSUE THREAD

**Issue by gussmith23** [gussmith23/glenside#65]
This is a hack for ISCA.

The basic idea is this: currently, I've implemented Relay-to-Glenside codegen for all of the operators in Resnet and Mobilenet. This means that the operators are actually functionally implemented in Glenside's syntax.

In the future, this will be good, because we'll need the operators in Glenside's syntax to do any reasoning about them. However, for right now, it's actually causing us more pain than good, for two reasons:
1. We don't actually tensorize these operators to any hardware yet. So there's no reason to expand them out into Glenside syntax.
2. Generating code from Glenside syntax produces really terrible code. We'd rather wrap the operators up as a single operator in Glenside and generate code using a handwritten kernel.

- [X] Add `RelayOperator`/`RelayOperatorCall` language constructs
- [ ] Add operators for each operator we care about
  + [x] BatchNormInference
  + [x] Softmax
  + [x] Relu
  + [x] Max pool 2d
  + [x] Global average pool 2d
  + [ ] Expand dims (maybe?)
  + [ ] Add
  + [ ] mul, div, negative, sqrt -- I don't think we need these, without SimplifyInference?
  + [x] Batch flatten
  + [x] Bias add
- [ ] Implement codegen for each operator
  + [ ] BatchNormInference
  + [ ] Softmax
  + [ ] Relu
  + [ ] Max pool 2d
  + [ ] Global average pool 2d
  + [ ] Expand dims (maybe?)
  + [ ] Add
  + [ ] mul, div, negative, sqrt -- I don't think we need these, without SimplifyInference?
  + [ ] Batch flatten
  + [ ] Bias add
